css/stanford_guides.css

```diff
@@ -33,4 +33,20 @@
     float: left;
     margin-left: -3em;
     margin-right: 25px;
-}
\ No newline at end of file
+}
+
+.guide-relates { 
+	background-color:#B6B1A9; margin:5px;
```

Comment:
- Tab indent should be changed to two spaces.
- Style declarations should be listed one per line. Please move the margin down a line.